vue-app/src/components/ContributionModal.vue

```diff
@@ -285,7 +297,7 @@ export default class ContributionModal extends Vue {
   margin: 1rem 0;
   margin-top: 1.5rem;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
```

Comment:
This is to keep the buttons on each side, not centered, keeping it consistent with the other modals.